libraries/cms/html/sortablelist.php

```diff
@@ -61,8 +61,8 @@ public static function sortable($tableId, $formId, $sortDir = 'asc', $saveOrderi
 		// Depends on jQuery UI
 		HTMLHelper::_('jquery.ui', array('core', 'sortable'));
 
-		HTMLHelper::_('script', 'legacy/sortablelist.min.js', false, true);
-		HTMLHelper::_('stylesheet', 'legacy/sortablelist.css', false, true, false);
+		HTMLHelper::_('script', 'legacy/sortablelist.min.js', ['version' => 'auto', 'relative' => true]);
+		HTMLHelper::_('stylesheet', 'legacy/sortablelist.css', ['version' => 'auto', 'relative' => true]);
```

Comment:
Should it not be 
```
		HTMLHelper::_('stylesheet', 'legacy/sortablelist.css');
```
```suggestion
		HTMLHelper::_('stylesheet', 'legacy/sortablelist.css', ['version' => 'auto', 'relative' => true]);
```
`HTMLHelper::_('stylesheet', 'legacy/sortablelist.css'

Reply:
I guess both will work? Did you test it?

Reply:
i did not test, just compared that line with similar changes in this pr.
i guess this should be normalised when globally modified.

Reply:
The old parameters do reflect `['version' => 'auto', 'relative' => true]` so I added it like that.